docs/install/install-from-packages.md

```diff
@@ -0,0 +1,279 @@
+Running sanity checks
+  - Python: ✓
+  - Node: ✓
+  - yo: ✓
+  - Docker: ✓
+  - git: ✓
+
+Summary:
+  - Plone version: 6.0.0b2
+  - Volto version: 16.0.0-alpha.35
+  - Output folder: /Users/stevepiercy/projects/Plone/documentation/ainstall/project-title
+
+Frontend codebase:
+ - Install latest @plone/generator-volto
+ - Generate frontend application with @plone/volto 16.0.0-alpha.35
+
+Backend codebase
+ - Format generated code in the backend
+================================================================================
+
+Project "Project Title" was generated
+
+Now, code it, create a git repository, push to your organization.
+
+Sorry for the convenience,
+The Plone Community.
+
+================================================================================
+```
+
+Change to your project directory {file}`project-title`.
+
+```shell
+cd project-title
+```
+
+Next we switch to using `make`.
+To see all available commands and their descriptions, enter the following command.
+
+```shell
+make help
+```
+
+To install both the Plone backend and frontend, use the following command.
+
+```shell
+make install
+```
+
+This will take a few minutes.
+☕️
+First the backend, then the frontend will be installed.
+At the start of the frontend installation part, you might see a prompt.
+
+```console
+Need to install the following packages:
+  mrs-developer
+Ok to proceed? (y)
+``` 
+
+Hit the {kbd}`Enter` key to proceed and install `mrs-developer`.
+
+When the process completes successfully, it will exit with a message similar to the following.
+
+```console
+✨  Done in 98.97s.
+```
+
+
+(install-packages-start-plone-label)=
+
+## Start Plone
+
+Plone 6 has two servers: one for the frontend, and one for the backend.
+As such, we need to maintain two active shell sessions, one for each server, to start your Plone site.
+
+
+(install-packages-start-plone-backend-label)=
+
+### Start Plone backend
+
+In the currently open session, issue the following command.
+
+```shell
+make start-backend
+```
+
+The Plone backend server starts up and emits messages to the console.
+
+```console
+Starting application
+2022-09-18 23:14:27,119 INFO    [chameleon.config:38][MainThread] directory cache: /path-to/my_project/project-title/backend/instance/var/cache.
+2022-09-18 23:14:30,812 INFO    [plone.volto:22][MainThread] Aliasing collective.folderish classes to plone.volto classes.
+2022-09-18 23:14:31,889 INFO    [Zope:42][MainThread] Ready to handle requests
+Starting server in PID 53838.
+2022-09-18 23:14:31,893 INFO    [waitress:486][MainThread] Serving on http://[::1]:8080
+2022-09-18 23:14:31,893 INFO    [waitress:486][MainThread] Serving on http://127.0.0.1:8080
+```
+
+At this point, you could visit the Plone Classic UI interface at the URL http://localhost:8080.
+However it is strongly recommended to use the new Plone frontend, Volto, to gain many advantages, including easier content editing and development of components.
+Continue to the next section to do so.
```

Comment:
I would put this in an info box or skip here at all. I fact we are telling what one could do, but better not do. This is for sure confusing.

Reply:
Good point. Deleted. The Volto frontend is almost in parity with the Classic UI, and that's the only reason I put it there in the first place. We can address any lack of parity by fixing the bugs instead of mentioning it in the docs.

Reply:
Deleted in 7c30797e40d105a901f5d6874b45bd9812da065c